src/main/java/Duke/DialogBox.java

```diff
@@ -0,0 +1,61 @@
+package Duke;
```

Comment:
Shouldn't this package name be in lower case, by the coding standards? I notice this in a couple other files too.